src/main/java/homework4/BusTicket.java

```diff
@@ -0,0 +1,70 @@
+package homework4;
+
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class BusTicket {
+    private TicketClass ticketClass;
+    private TicketType ticketType;
+    private LocalDateTime startDate;
+    private int pprice;
```

Comment:
pprice?
Check in other places as well please

Reply:
Yes, it's because i have price in usual ticket. and when i search field price in validator it doesnt work i need change

Reply:
check the name..))